instrumentation/micrometer/micrometer-1.5/javaagent/build.gradle.kts

```diff
@@ -0,0 +1,21 @@
+plugins {
+  id("otel.javaagent-instrumentation")
+}
+
+muzzle {
+  pass {
+    group.set("io.micrometer")
+    module.set("micrometer-core")
+    versions.set("[1.5.0,)")
+    assertInverse.set(true)
+  }
+}
+
+dependencies {
+  library("io.micrometer:micrometer-core:1.5.0")
```

Comment:
This is a pretty old version, it is not supported anymore, I would go with `1.8.x` (latest).

Reply:
This is a compile dependency, users bring in the version that they want. Our standard practice is to target the lowest version we support as the compile dependency, while running tests on that and the latest version

Reply:
I get that but do you want to support a version that reached its end of life and nobody should use it in prod?
Currently the oldest version that we support is the 1.7.x line which will reach its EOL in May.

Reply:
We will support all versions from 1.5 upwards, 1.8 included. As Anuraag mentioned, we generally aim to support the lowest version possible - the javaagent is often attached to applications that haven't been updated for some time; and it is supposed to work with no code changes (like upgrading the micrometer dependency version) at all.